tests/transformers/test_numerical.py

```diff
@@ -33,6 +33,114 @@ def test_fit(self):
         assert transformer.null_transformer.fill_value == expect_fill_value
         assert transformer._dtype == expect_dtype
 
+    def test_fit_min_max_none(self):
+        """Test fit min and max parameters with ``None``
+
+        If the min and max parameters are set to ``None``,
+        the ``fit`` method should not set its ``min`` or ``max``
+        instance variables.
+        """
+        # Setup
+        data = np.array([1.5, None, 2.5])
+
+        # Run
+        transformer = NumericalTransformer(dtype=np.float, nan='nan',
+                                           min_value=None, max_value=None)
+        transformer.fit(data)
+
+        # Asserts
+        assert transformer.min_value is None
+        assert transformer.max_value is None
+
+    def test_fit_min_max_int(self):
+        """Test fit min and max parameters with int values
+
+        If the min and max parameters are set to an int,
+        the ``fit`` method should not change them.
+        """
+        # Setup
+        data = np.array([1.5, None, 2.5])
+
+        # Run
+        transformer = NumericalTransformer(dtype=np.float, nan='nan',
+                                           min_value=1, max_value=10)
+        transformer.fit(data)
+
+        # Asserts
+        assert transformer.min_value == 1
+        assert transformer.max_value == 10
+
+    def test_fit_min_max_auto(self):
+        """Test fit min and max parameters with ``'auto'``
+
+        If the min or max parameters are set to ``'auto'``
+        the ``fit`` method should learn them from the
+        fitted data.
+        """
+        # Setup
+        data = np.array([-100, -5000, 0, 100, 4000])
+
+        # Run
+        transformer = NumericalTransformer(dtype=np.float, nan='nan',
+                                           min_value='auto', max_value='auto')
+        transformer.fit(data)
+
+        # Asserts
+        assert transformer.min_value == -5000
+        assert transformer.max_value == 4000
+
+    def test_reverse_transform_min_no_max(self):
```

Comment:
Maybe we should add tests with null values? The null values need to be passed by making the input a 2d matrix with a second column that has values from 0 to 1 with at least 1 value above 0.5